pending/xnviewmp.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2018 Andrew Ziem
+    https://www.bleachbit.org
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	CleanerML written by https://github.com/Tobias-B-Besemer
+	v0.1 done on 2018-12-19
+
+-->
+<cleaner id="xnviewmp">
+  <label>XnView MP for Windows</label>
+  <description>Picture Viewer</description>
+  <option id="customPath">
+    <label>Custom Path of XnView MP</label>
+    <description>Delete the custom path of XnView MP</description>
+        <action command="ini" search="glob" path="$APPDATA\XnViewMP\xnview.ini" name="customPath"/>
+        <action command="ini" search="glob" path="%ProgramFiles%\XnViewMP\xnview.ini" name="customPath"/>
+        <action command="ini" search="glob" path="%ProgramFiles(x86)%\XnViewMP\xnview.ini" name="customPath"/>
+        <action command="winreg" path="HKCU\Software\XnView\XnViewMP\Start\customPath" />
+  </option>
+  <option id="savefolder">
+    <label>Save Folder of Viewer</label>
+    <description>Delete the Save Folder of Viewer</description>
+        <action command="ini" search="glob" path="$APPDATA\XnViewMP\xnview.ini" name="saveFolder"/>
```

Comment:
glob is not needed in several places, including this line, unless there is a wildcard

Reply:
Can I use command="ini" search="file" ???

Reply:
Yeah, I have used command="ini" search="file" e.g. in total_commander2.xml...

Btw.: You don't need to check in my PR, if they aren't perfect! Makes just work for you, when I make some hours later the next PR to make it hopefully this time perfect... ;-)

Reply:
Yes, any command that works on a file (such as ini, json, delete)  works with any file search (such as file, glob, walk.all, walk.files).

You wrote "you don't need to check in my PR." Ideally I would not leave open any issues or PRs, but I do my best to process them. I do not like issues or pull requests staying open for a long time, so in cases such as this (which is not a functional problem) I try to not be too picky.

Reply:
P.S., thanks for your attention to detail

Reply:
No prob! I wanna make them perfect! ;-)